### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by stefonarch**
Fixed one error and added link to weblate.

### CODE REVIEW

**Comment by stefonarch** [comment]
Added a note about panel and config. Any objections merging this?

**Comment by stefonarch** [comment]
Thanks!

**Comment by stefonarch** [comment]
Actually this is not true yet until @tsujan makes the change to libfm-qt:

> * `lxqt-applications-fm.menu`:
>   * Application chooser menu in PCmanFm-qt (right click menu "Open with → Other applications")
>   * Applications menu in PcManFm-qt if installed outside of LXQt.

**Comment by tsujan** [comment]
> until @tsujan makes the change to libfm-qt:

First I should test the changes in lxqt-panel and lxqt-config; I'm doing it now....

**Comment by tsujan** [comment]
All done smoothly (by removal, instead of overwriting) and everything seems fine — although I still have `lxmenu-data` (will remove it after patching `libfm-qt`).

**Comment by tsujan** [comment]
Isn't this better?

`Default menu files for LXQt Panel, Configuration Center and PCManFM-Qt/libfm-qt.`

Also, notice the difference in "PCManFM-Qt" and also "files" instead of "file.

**Comment by stefonarch** [comment]
I think we need a wiki for "Capitalized characters of LXQt" :)

I just noticed that the "About" on top right is the one I choose years ago and should be improved, I've no "wheel" to click there to change this text here. Original:

>freedesktop.org application menu definition files 

What about "Freedesktop.org compliant menu files for LXQt Panel, Configuration Center and PCManFM-Qt/libfm-qt." ? I dropped "default" as it contains also 2 other application menus.

File: README.md [hunk 1]
```diff
@@ -1,20 +1,21 @@
+
 # lxqt-menu-data
 
-Provides translations for menu categories and the menu files for LXQt and PCmanFm-qt.
+Freedesktop.org compliant menu files for LXQt Panel, Configuration Center and PCManFM-Qt/libfm-qt.
 
-Runtime dependency for [lxqt-panel](https://github.com/lxqt/lxqt-panel) and [pcmanfm-qt](https://github.com/lxqt/pcmanfm-qt).
+Runtime and build dependency for [lxqt-panel](https://github.com/lxqt/lxqt-panel), [lxqt-config](https://github.com/lxqt/lxqt-config) and [pcmanfm-qt](https://github.com/lxqt/pcmanfm-qt).
 
 Replacement for and partially based of [lxmenu-data](https://github.com/lxde/lxmenu-data) (LXDE).
```
Comment (tsujan) [inline README.md:8]: "based of" → "based on"

File: README.md [hunk 2]
```diff
@@ -1,20 +1,21 @@
...
 
 ### Menu Files
 
-* `lxqt-applications.menu`: Default menu file for the "Main Menu plugin" in lxqt-panel and "Applications" in PcManFm-qt.
+* `lxqt-applications.menu`: Default menu file for the "Main Menu" plugin in lxqt-panel and "Applications" in PcManFm-qt.
```
Comment (tsujan) [inline README.md:12]: PcManFm-qt → PCManFM-Qt

The rest seems good.

**Pull request merged**